core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -215,7 +216,11 @@ private synchronized void initServer() {
         .withHeartbeatInterval(Duration.ofMillis(mConf.getHeartbeatIntervalMs()))
         .withSnapshotAllowed(mSnapshotAllowed)
         .withSerializer(createSerializer())
-        .withTransport(new NettyTransport())
+        .withTransport(new CopycatGrpcTransport(
```

Comment:
This is to support gRPC auth scenarios fully, in case we need to plug a copyat client in a non-master process. I think it's good to at least internally separate client/server configuration/user. 

Will implement a convenience constructor that supplies existing one with duplicates.